src/Logary/MiddlewareModule.fs

```diff
@@ -65,6 +65,7 @@ module Middleware =
 
   let useAmbientSpanId () : Middleware =
```

Comment:
What difference is it to a consumer if it's a `span` or a `spanId`? Doesn't "use global span id" imply "use global span"? What's a global span? If I run a web app with 100 concurrent requests on Suave, what's the global span?